store-implementation/accumulo-store/src/test/resources/empty-store.properties

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2016-2020 Crown Copyright
```

Comment:
Any new files you create (this one, RFileReaderIteratorTest.java and AccumuloTabletTest.java should have just the current year in the copyright. For example
```suggestion
# Copyright 2020 Crown Copyright
```

Reply:
Thanks and done